game/graphics/objectsbucket.cpp

```diff
@@ -363,7 +363,7 @@ void ObjectsBucket::uboSetSkeleton(Descriptors& v, uint8_t fId) {
 void ObjectsBucket::uboSetDynamic(Descriptors& v, Object& obj, uint8_t fId) {
   auto& ubo = v.ubo[fId][SceneGlobals::V_Main];
 
-  if(mat.frames.size()!=0) {
+  if(mat.frames.size()!=0 && mat.texAniFPSInv != 0) {
```

Comment:
Just curious: were the issues with `mat.texAniFPSInv==0` ?

Reply:
Yeah I had divide by 0 errors. I couldn't figure out why at the time and it still doesn't make sense to me.